packages/trader/src/App/Components/Elements/TogglePositions/__tests__/toggle-positions-mobile.spec.jsx

```diff
@@ -0,0 +1,181 @@
+};
+
+describe('<TogglePositionsMobile />', () => {
+    const mockTogglePositionsMobile = (mocked_store, mocked_props) => {
+        return (
+            <TraderProviders store={mocked_store}>
+                <TogglePositionsMobile {...mocked_props} />
+            </TraderProviders>
+        );
+    };
+
+    beforeAll(() => {
+        ReactDOM.createPortal = jest.fn(component => {
+            return component;
+        });
+    });
+
+    afterAll(() => {
+        ReactDOM.createPortal.mockClear();
+    });
+
+    it('should render <TogglePositionsMobile /> with <TogglePositions /> and <Modal /> content if is_positions_drawer_on === true', () => {
+        const mock_root_store = mockStore({ ...default_mock_store });
+
+        render(mockTogglePositionsMobile(mock_root_store, { ...default_mocked_props }));
+
+        screen.getAllByText(/test icon/i).forEach(icon => expect(icon).toBeInTheDocument());
+        screen.queryAllByAltText(/position modal card/i).forEach(card => expect(card).toBeInTheDocument());
+        expect(screen.getByTestId(/dt_positions_toggle/i)).toBeInTheDocument();
+        expect(screen.getByText(/recent positions/i)).toBeInTheDocument();
+        expect(screen.getByText(/go to reports/i)).toBeInTheDocument();
+    });
+    it('should not render <TogglePositionsMobile /> with <TogglePositions /> and <Modal /> content if is_positions_drawer_on === false', () => {
+        const mock_root_store = mockStore({ ...default_mock_store, ui: { is_positions_drawer_on: false } });
+
+        render(mockTogglePositionsMobile(mock_root_store, { ...default_mocked_props }));
+
+        screen.queryAllByAltText(/position modal card/i).forEach(card => expect(card).not.toBeInTheDocument());
+        expect(screen.queryByText(/recent positions/i)).not.toBeInTheDocument();
+        expect(screen.queryByText(/go to reports/i)).not.toBeInTheDocument();
+    });
+    it('should render <EmptyPortfolioMessage /> with error text if is_empty === true', () => {
+        const mock_root_store = mockStore({ ...default_mock_store });
+
+        render(mockTogglePositionsMobile(mock_root_store, { ...default_mocked_props, is_empty: true }));
+
+        expect(screen.getByText(/test error/i)).toBeInTheDocument();
+    });
+    it('should call function togglePositionsDrawer if <NavLink /> was clicked', () => {
+        const mock_root_store = mockStore({ ...default_mock_store });
+
+        render(mockTogglePositionsMobile(mock_root_store, { ...default_mocked_props }));
+        const nav_link_button = screen.getByText(/go to reports/i);
+        userEvent.click(nav_link_button);
+
+        expect(mock_root_store.ui.togglePositionsDrawer).toBeCalled();
+    });
+    it('should call function togglePositionsDrawer if close button in positions-modal was clicked', () => {
+        const mock_root_store = mockStore({ ...default_mock_store });
+
+        render(mockTogglePositionsMobile(mock_root_store, { ...default_mocked_props }));
```

Comment:
@kate-deriv 
`{ ...default_mocked_props }` is the same as `default_mocked_props` :) we don't need to use spread operator here, right?